languages/common-lisp/exercises/concept/basics/.docs/instructions.md

```diff
@@ -0,0 +1,17 @@
+In this exercise, you fill need to fill in a number of incomplete functions.
```

Comment:
Each exercise should have a small story or theme, just to give the tasks some context. Similarly, each task should show an example, see [this example](https://github.com/exercism/v3/blob/csharp/exercises/basics/languages/csharp/exercises/concept/basics/.docs/instructions.md). That said, it can be tricky to do so, as this exercise does introduce a lot of things. I'd argue that perhaps this exercise introduces _too_ many things. For example, the truthy/falsy bit might be a perfect candidate to extract to a separate exercise (which can also introduce boolean logic operators and such). We already have a booleans exercises implemented in several languages, including one in another functional language: [F#](https://github.com/exercism/v3/tree/master/languages/fsharp/exercises/concept/booleans).

The way I look at the basics exercise, is to introduce just the bare basics. For C# and F# that includes: assignment of "variables", defining functions (with zero or more parameters), returning values from functions, calling a function and comments. As functions cannot be interesting without using a data type, C# and F# use integers (just one or more consecutive digits) and basic math operators, as their notation is usually the same as the regular mathematical notation which everyone will know.

Does this help a bit?

Reply:
This helps a ton! Thank you very much! The `truthy-and-falsy` concept was originally part of the `conditionals` concept, so I may try to defer it back to that one. The only reason that I moved it up was because the functions for testing if an S-Expression is an atom or cons return truthy and falsy values.

I think I might try to push that back to the `conditionals` concept (probably the third one) and add in a bit more of a story here!

I think I'll try to stick to symbols as my datatypes here, as arithmetic in Common Lisp, while nothing terribly strange, does look a bit different (I have `numbers.basic` exercise lined up for the second exercise).

Reply:
Sounds good!

Reply:
@ErikSchierboom the new commit has some fun story added to it and also simplifies a little by putting off the true and false discussion. I would love to know what you think!

Reply:
I really love the story and the simplification! I think it makes the exercise more focused and better. Awesome work.